release_notes/ose_3_2_release_notes.adoc

```diff
@@ -0,0 +1,736 @@
+use `*` to allow all volumes.
+** By default, regular users are now forbidden from directly mounting any of the
+remote volume type; they must use a persistent volume claim (PVC).
+- The new `*ReadOnlyRootFilesystem*` field in SCCs allows an administrator to
+force containers to run with a read-only root file system.
+** If set to true, containers are required to run with a read-only root file system
+by their `*SecurityContext*`. Containers that do not set this value to true will
+be defaulted. Containers that explicitly set this value to false will be
+rejected.
+** If set to false, containers may use a read-only root file system, but they are
+not forced to run with one.
+- By default, the *restricted* and *anyuid* SCCs drop Linux capabilities that
+could be used to escalate container privileges. Administrators can change the
+list of default or enforced capabilities.
+- A constant-time string comparison is now used on webhooks.
+- Only users authenticated via OAuth can request projects.
+- A GitLab server can now be used as an identity provider. See
+link:../install_config/configuring_authentication.html#GitLab[Configuring
+Authentication] for details.
+- The `SETUID` and `SETGID` capabilities have been added back to the *anyuid* SCC,
+which ensures that programs that start as root and then drop to a lower
+permission level will work by default.
+- Quota support has been added for `*emptydir*`. When the quota is enabled on an
+XFS system, nodes will limit the amount of space any given project can use on a
+node to a fixed upper bound. The quota is tied to the `*FSGroup*` of the
+project. Administrators can control this value by editing the project directly
+or allowing users to set `*FSGroup*` via SCCs.
+- The `*DaemonSet*` object is now limited to cluster administrators because pods
+running under a `*DaemonSet*` are considered to have higher priority than
+regular pods, and for regular users on the cluster this could be a security
+issue.
+- Administrators can prevent clients from accessing the API by their `User-Agent`
+header the new `*userAgentMatching*` configuration setting.
+
+==== Integrated Docker Registry
+
+- The integrated Docker registry now supports Azure Blob Storage, OpenStack Swift,
+and Amazon CloudFront as storage back ends.
+- A readiness probe and health check have been added to the integrated registry to
+ensure new instances do not serve traffic until they are fully initialized.
+
+==== Routes
+
+- You can limit the frequency of router reloads using the `--interval=DURATION`
+flag or `*RELOAD_INTERVAL*` environment variable to the router. This can
+minimize the memory and CPU used by the router while reloading, at the cost of
+delaying when the route is exposed via the router.
+- Routers now report back status to the master about whether routes are accepted,
+rejected, or conflict with other users. The CLI will now display that error
+information, allowing users to know that the route is not being served.
+
```

Comment:
Yeah would be good to add something. Ala:
`Routers now enable you to specify a selection criteria for either namespaces or labels on routes. This enables you to select the routes a router would expose and you can use this functionality to distribute routes across a set of routers (shards).`
You may want to clean that up though.